### DISCUSSION TITLE
Qexo能加强支持一下html语法吗

### DISCUSSION THREAD

**Discussion by haloged** [Qexo/Qexo#398] [Ideas]
Qexo能加强一下支持html代码吗，写个dev标签就会直接隐藏掉，被迫直接去github改源文件

**Comment by am-abudu** [comment]
有三个选择
1. 等待Vditor添加该特性(几乎不可能)
2. 需要另外的更棒的编辑器(如果你有，请告诉我)
3. 额外支持兼容编辑器(AceEditor, 和配置编辑一样)

**Reply by haloged** [reply]
可不可以用editor.md呢，这个对html的支持还挺好的（小白勿喷）

**Reply by am-abudu** [reply]
有报告称Editor.md对于Safari的支持不佳，望举证

**Reply by haloged** [reply]
那stackedit呢(https://gitee.com/mafgwo/stackedit)

**Reply by am-abudu** [reply]
这应该不是一个通用的编辑器，可能不是很合适
